src/parser.rs

```diff
@@ -378,6 +378,10 @@ where
                         self.expect_ident();
                     }
 
+                    while self.peek() != Some(TOKEN_SEMICOLON) {
+                        self.parse_expr();
```

Comment:
I believe I tricked you into writing this, I forgot `parse_expr` handles functions and we don't want `inherit a b` to be a function call. Changing it to just `parse_val` should work, though :)